Remotion/ObjectBinding/Web.Test/TestForm.aspx

```diff
@@ -14,49 +14,60 @@
  % You should have received a copy of the GNU Lesser General Public License
  % along with re-motion; if not, see http://www.gnu.org/licenses.
 --%>
-
 <%@ Page Language="c#" CodeBehind="TestForm.aspx.cs" AutoEventWireup="false" Inherits="OBWTest.TestForm" %>
 
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
   <title>Test Form</title>
   <remotion:HtmlHeadContents ID="HtmlHeadContents" runat="server" />
-  <script type="text/javascript">
+</head>
+<body>
+  <form id="form1" runat="server">
+  <asp:ScriptManager ID="ScriptManager" runat="server" EnablePartialRendering="true"
+    AsyncPostBackTimeout="3600" />
+  <input id="StringField" type="text" />
+  <input id="IntField" type="text" />
+  <input type="button" value="ASP.NET Ajax" onclick="DoAspNetAjaxCall()" />
+  <input type="button" value="jQuery Ajax" onclick="DoJQueryAjaxCall()" />
+  <div id="Result">
+  </div>
+  </form>
+<script type="text/javascript">
     function DoAspNetAjaxCall()
     {
-      $("#Result").text("");
+      document.getElementById("Result").textContent = "";
 
-      var intValueAsString = $("#IntField").val();
+      var intValueAsString = document.getElementById("IntField").value;
       intValueAsString = intValueAsString == '' ? 0 : intValueAsString;
       var params = {
-        stringValue: $("#StringField").val(),
+        stringValue: document.getElementById("StringField").value,
         intValue: parseInt(intValueAsString)
       };
       executingRequest = Sys.Net.WebServiceProxy.invoke("TestService.asmx", "DoStuff", false, params,
                                           function (result, context, methodName)
                                           {
                                             executingRequest = null;
-                                            $("#Result").text(result);
+                                            document.getElementById("Result").textContent = result;
```

Comment:
in the other scripts, we're using innerText, here it's textContent, why?

Reply:
I looked up the difference and there seems to be no difference when setting the properties, but `textContent` is semantically more accurate when setting the text content of a element. `innerText` removes invisible elements and such when getting the property (a rendered version of `textContent`) and this is sometimes desired.

Reply:
ah, I.C. So, jQuery text() behaves more like textContent? I never new that one. However, I don't have problems with leading whitespace, so the question is, should we stick with innerText to have everything consistent and do a separate task for reviewing/cleaning up or accept that it's inconsistent right now?

Reply:
Since `textContent` is available on `Node` instead of `HTMLElement` it seems to be the better choice (when setting - getting always depends on the usage). But I don't think it is worth fixing, we only have 3 usages of `innerText` that would need to be fixed.